changelog/139.breaking.rst

```diff
@@ -0,0 +1,13 @@
+A series of functions were added from `sunkit_image.asda` by declassing the code.
+Previously, if one has to make objects of class ``Asda`` and ``Lamb_Oseen`` to implement asda.
+Now, all the functions are now declassed, now no object creation is required
+to implement any functionality of asda.
```

Comment:
How does one use asda now that its been declassed? That needs to be in this changelog.

Reply:
Ohhk... I add that in more detail